app/hotels/src/filter/price/PricePopup.js

```diff
@@ -49,36 +56,35 @@ export default class PricePopup extends React.Component<Props, State> {
       maxPrice: this.state.price.end,
     });
 
-  renderLabel = () => {
-    const { start, end } = this.state.price;
-    const { currency } = this.props;
-
-    return (
-      <Text>
-        <Translation passThrough={`${start} ${currency}`} />
-        <Translation passThrough=" - " />
-        <Translation passThrough={`${end} ${currency}`} />
-      </Text>
-    );
-  };
-
   render() {
     const { start, end } = this.state.price;
-    const { min, max } = this.props;
+    const { min, max, currency } = this.props;
 
-    const label = this.renderLabel();
     return (
       <ButtonPopup
         buttonTitle={<Translation id="hotels_search.filter.price_popup.save" />}
+        buttonCloseTitle={
+          <Translation id="hotels_search.filter.hotel_facilities_popup.close" />
+        }
         onSave={this.onSave}
         onClose={this.props.onClose}
         isVisible={this.props.isVisible}
       >
-        <Text>
+        <Text style={styles.title}>
           <Translation id="hotels_search.filter.price_popup.title" />
           <Translation passThrough=" " />
-          {label}
+          <Text style={styles.subtitle}>
+            <Translation id="hotels_search.filter.price_popup.subtitle" />
+          </Text>
         </Text>
+        <SliderLabels
+          max={max}
+          min={min}
+          startValue={start}
+          endValue={end}
+          startLabel={<Translation passThrough={`${start} ${currency}`} />}
```

Comment:
This is not a good idea: I think you should make it a translation id with two variables:
```
'new.translation.id': '__1_amount__ __2_currency__'
```
and then use it as
```
<Translation id='new.translation.id' values={{amount: "23", currency: "EUR"}} />
```

The reason is that in other languages, the currency sign can be in front, like in English: $20.

This solution is still not perfect if we had to deal with prices over 1,000, as you would need to add a comma in English, but not in French where it would be a space (1 000) for instance. Same with decimal values.